RecoJets/JetProducers/plugins/BoostedTauSeedsProducer.cc

```diff
@@ -265,25 +289,25 @@ void BoostedTauSeedsProducer::produce(edm::Event& evt, const edm::EventSetup& es
       selectedSubjetPFCandidateAssociationForIsolation->insert(subjetRef2, pfCandidate);
     }
 
-    // find all PFCandidates that are constituents of the **other** subjet
-    std::vector<reco::PFCandidateRef> pfCandidatesInSubjet1 = getPFCandidates_exclJetConstituents(pfCandidates, subjetConstituents2, 1.e-4, true);
-    std::vector<reco::PFCandidateRef> pfCandidatesInSubjet2 = getPFCandidates_exclJetConstituents(pfCandidates, subjetConstituents1, 1.e-4, true);
-    if ( verbosity_ >= 1 ) {
-      std::cout << "#pfCandidatesInSubjet1 = " << pfCandidatesInSubjet1.size() << std::endl;
-      std::cout << "#pfCandidatesInSubjet2 = " << pfCandidatesInSubjet2.size() << std::endl;
-    }
-
-    BOOST_FOREACH( const reco::PFCandidateRef& pfCandidate, pfCandidatesInSubjet1 ) {
-      selectedSubjetPFCandidateAssociationForIsoDepositVetos->insert(subjetRef1, pfCandidate);
-    }
-    BOOST_FOREACH( const reco::PFCandidateRef& pfCandidate, pfCandidatesInSubjet2 ) {
-      selectedSubjetPFCandidateAssociationForIsoDepositVetos->insert(subjetRef2, pfCandidate);
-    }
+    //// find all PFCandidates that are constituents of the **other** subjet
+    //std::vector<reco::PFCandidateRef> pfCandidatesInSubjet1 = getPFCandidates_exclJetConstituents(pfCandidates, subjetConstituents2, 1.e-4, true);
+    //std::vector<reco::PFCandidateRef> pfCandidatesInSubjet2 = getPFCandidates_exclJetConstituents(pfCandidates, subjetConstituents1, 1.e-4, true);
+    //if ( verbosity_ >= 1 ) {
+    //  std::cout << "#pfCandidatesInSubjet1 = " << pfCandidatesInSubjet1.size() << std::endl;
+    //  std::cout << "#pfCandidatesInSubjet2 = " << pfCandidatesInSubjet2.size() << std::endl;
+    //}
+    //
+    //BOOST_FOREACH( const reco::PFCandidateRef& pfCandidate, pfCandidatesInSubjet1 ) {
+    //  selectedSubjetPFCandidateAssociationForIsoDepositVetos->insert(subjetRef1, pfCandidate);
+    //}
+    //BOOST_FOREACH( const reco::PFCandidateRef& pfCandidate, pfCandidatesInSubjet2 ) {
+    //  selectedSubjetPFCandidateAssociationForIsoDepositVetos->insert(subjetRef2, pfCandidate);
+    //}
```

Comment:
We recompute the isolation for electron and muons in the "private" analysis code starting from PF isolation and removing the PF candidates of the other reconstructed and identified taus. 
Besically here we just need to remove from each subjet the candidates of the other subjet and then used the subjet for the tau reconstruction. We don't need to save for each subjet also the candidates of the other because they can be selected from the candidates of the other subjet afterwards.  In fact we didn't use this collection even in Run I, but if you think that we need this code we can keep it.